objmodel/chapter.md

```diff
@@ -1,86 +1,86 @@
+To understand the behaviour and the design choices better, the chapter will also
+present tests for the object model. The tests can be run with either py.test or nose.
 
-The object models presented in this chapter will also be implemented in Python.
-The code works on both Python 2.7 and 3.4. The tests can be run with either
-py.test or nose.
-To understand the behavior and the design choice better the chapter will also
-present tests for the object model. The choice of Python as an implementation
+The choice of Python as an implementation
 language is quite unrealistic. A "real" VM is typically implemented in a
-low-level language like C/C++ and needs a lot of additional attention to
+low-level language like C/C++ and needs a lot of attention to
 engineering detail to make it efficient. However, the simpler implementation
 language makes it easier to focus on actual behaviour differences instead of
 getting bogged down by implementation details.
 
 
 
-Method based model
-----------------------
+## Method-Based Model
 
 The object model we will start out with is an extremely simplified version of
 that of Smalltalk. Smalltalk was an object-oriented programming language
-designed by Alan Kay's group at Xerox Parc in the 1970s. It popularized
-object-oriented programming and is the source of many features found in today's
+designed by Alan Kay's group at Xerox PARC in the 1970s. It popularized
+object-oriented programming, and is the source of many features found in today's
 programming languages. One of the core tenets of Smalltalk's language design
 was "everything is an object". Smalltalk's most immediate successor in use
 today is Ruby, which uses a more C-like syntax but retains most of Smalltalk's
 object model.
 
-The object model of this section will have classes and instances of them, the
+The object model in this section will have classes and instances of them, the
 ability to read
-and write attributes into objects, the ability to call methods on objects and
+and write attributes into objects, the ability to call methods on objects, and
 the ability for a class to be a subclass of another class. Right from the
-beginning, classes will be completely regular objects, that can themselves have
+beginning, classes will be regular objects that can themselves have
 attributes and methods.
-A note on terminology: In this chapter I will use the word "instance" to mean
-"an object that is not a class".
 
-To start the implementation a good approach is to write a test to think about
+A good approach to start with is to write a test to specify
 what the to-be-implemented behaviour should be. All tests presented in this
-chapter will always consist of two parts: A bit of completely regular Python
-code defining and using a few classes, making use of increasingly advanced
-features of the Python object model. The second half of each test is the
+chapter will consist of two parts. First, a bit of regular Python
```

Comment:
When you introduce something as having a number of parts, it's helpful to the reader if you are very clear about what those parts are as you come to them.